app/Http/Controllers/JournalController.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Request\JournalRequest;
```

Comment:
Change `use App\Http\Request\JournalRequest;`  to `use App\Http\Requests\JournalRequest;`. There is typo in `Request` namespace.